The list of runtime versions that fixed the quadratic-time issue in HTMLParser
needs to be adjusted because we backported the fix to older versions.

--- django-5.2.6/tests/utils_tests/test_html.py
+++ django-5.2.6/tests/utils_tests/test_html.py
@@ -126,9 +126,9 @@ class TestUtilsHtml(SimpleTestCase):
         # https://github.com/python/cpython/commit/6eb6c5db
         min_fixed = {
             (3, 14): (3, 14),
-            (3, 13): (3, 13, 6),
+            (3, 13): (3, 13, 4),
             (3, 12): (3, 12, 12),
-            (3, 11): (3, 11, 14),
+            (3, 11): (3, 11, 13),
             (3, 10): (3, 10, 19),
             (3, 9): (3, 9, 24),
         }
